@xen-orchestra/fs/src/abstract.js

```diff
@@ -54,7 +54,7 @@ export default class RemoteHandlerAbstract {
 
   async test (): Promise<Object> {
     const testFileName = `${Date.now()}.test`
-    const data = getPseudoRandomBytes(1024 * 1024)
+    const data = await fromCallback(cb => randomBytes(1024 * 1024, cb))
```

Comment:
`randomBytes` is much slower than `getPseudoRandomBytes` (5-10× slower in latest tests), but it probably does not matter for so little data.